tools/scalar-schema/project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject scalar-schema "3.0.0"
+(defproject scalar-schema "4.0.0-SNAPSHOT"
```

Comment:
Any reason for `4.0.0-SNAPSHOT`?

Reply:
This is because the `master` branch is for the next release, `4.0.0`.

Similarly, I changed the version to `3.0.2-SNAPSHOT` in the `3.0` branch, and I'm going to change the versions to `3.2.0-SNAPSHOT` and `3.1.1-SNAPSHOT` in the `3` and `3.1` branches, respectively.

I think the version should be the next release version in the branch + `-SNAPSHOT`.

Reply:
Oh, I didn't know that's the way it is. Thank you!